src/components/AddEventPage.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { connect } from "react-redux";
+import EventForm from "./EventForm";
+import { addEvent } from "../actions/events";
+
+export class AddEventPage extends React.Component {
+  onSubmit = event => {
+    this.props.addEvent(event);
+    this.props.history.push("/");
+  };
+
+  render() {
+    return (
+      <div>
+        addpage
+        <EventForm onSubmit={this.onSubmit} />
+      </div>
+    );
+  }
+}
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
You can omit `mapDispatchToProps` and use shorter way of connecting redux actions with components. Consider example below:
```
      </div>
    );
  }
}

 export default connect(
  undefined,
 { addEvent },
)(AddEventPage);
```